src/emulate.c

```diff
@@ -1456,3 +1457,13 @@ void ecall_handler(riscv_t *rv)
     rv_except_ecall_M(rv, 0);
     syscall_handler(rv);
 }
+
```

Comment:
I feel a little bit weird to dump the result at STDOUT and forcing to silence the output from `syscall_handler` for this. Since these two features don't depend on each other. Will providing an extra argument for option `--dump-registers` to specify where to output(it could be a file name or STDOUT, if we really need it) become better?

And if we would like to not output the message from  `syscall_handler`, we may introduce another command line option there.